packs/data/spell-effects.db/spell-effect-blessing-of-defiance.json

```diff
@@ -31,15 +31,84 @@
                         "value": "will"
                     }
                 ],
-                "flag": "blessingOfDefiance",
+                "flag": "blessingOfDefianceSave",
```

Comment:
If you also add:
`"rollOption": "blessing-of-defiance-save"`
You shouldn't need the RollOption RE.